paasta_tools/setup_tron_namespace.py

```diff
@@ -62,22 +75,49 @@ def main():
     else:
         logging.basicConfig(level=logging.WARNING)
 
-    try:
-        new_config = tron_tools.create_complete_config(
-            service=args.service,
-            soa_dir=args.soa_dir,
-        )
+        if args.all_namepsaces:
+            tron_config = tron_tools.load_tron_config()
+            try:
+                services = tron_tools.get_tron_namespaces_for_cluster(
+                    cluster=tron_config.get_tron_cluster())
```

Comment:
`get_tron_namespaces_for_cluster` already defaults to the cluster in the tron config, so you don't have to load the tron config.